### PR TITLE
Drop pims dependency

### PR DESCRIPTION
**Pull request by nkeim**
This addresses #491.

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -32,7 +31,6 @@ requirements:
     - numexpr  # required for pandas DataFrame.query, used in examples
```
Comment (caspervdw) [inline conda-recipe/meta.yaml]: The conda recipe can be dropped altogether, it's an old version, the new one is maintained at conda-forge

**Comment by nkeim** [comment]
Aha! @caspervdw I suppose we will take care of conda-forge when we do the release?

**Comment by nkeim** [comment]
Ah! I wasn't looking for the other pims imports.

- The use of pims in `plots.py` is an enhancement, and the `ImportError` is already handled; we just need to update the error message.
- I don't see pims imported anywhere in `refine/least_squares.py`; there are only references to pims in the docstrings, and the use of pims objects as arguments is optional.
- In `artificial.py`, the only serious use of pims is the `CoordinateReader` class, which is used for many `find_link` tests. It seems like making a pims-free equivalent would be reinventing the wheel. Perhaps we could move this class to `tests/common.py`, and skip most/all `find_link` tests if pims is missing?

If we want to release v0.4.1 very soon, I think it would be reasonable to omit this PR.

**Comment by caspervdw** [comment]
@nkeim  I did some work on this in the train this morning. I think `artificial.py` can function fine without PIMS. Also, for testing purposes, I created a `tests.common.TrackpyImageSequence` that reads images via matplotlib `imread`.

See https://github.com/nkeim/trackpy/pull/7

**Pull request merged**